data/input_files/tests/nonlinear_solid/serac_component_bc/qs_attribute_solve.lua

```diff
@@ -0,0 +1,55 @@
+-- Comparison information
+expected_l2norm = 0.03330115
```

Comment:
I can see this eventually moving to an integrated test solution. @white238 do you think it's best to put it in an input deck for now? It definitely improves readability.

Reply:
Yes we should be moving as many of these to input decks as we can.  It will assist in the integrated tests as well.